src/website/page.gleam

```diff
@@ -395,6 +395,9 @@ pub fn news_post(post: news.NewsPost, ctx: site.Context) -> fs.File {
                 <> post.path
                 <> "')",
             ),
+            attr.data("tooltip", "Copied the post URL!"),
+            attr.data("tooltip-position", "right"),
+            attr.data("tooltip-toggle", "click"),
```

Comment:
It says toggle but it looks more like a trigger.

Could get rid of this? idk if there's triggers other than click?

Reply:
The reason for this is to make it clear that we can implement different kinds of tooltips, I have hover-activated ones in mind for other pages. If you think it's a bit premature to do this I'm happy to simplify it